_content/tour/por/slices.article

```diff
@@ -102,17 +102,20 @@ to move the data around my program. This includes declaring fields on a type.
         Z []bool
     }
 
-This goes for all of Go’s internal data structures (slices, maps, channels, interfaces, and functions).
+Isso se aplica a todas as estruturas de dados internas do Go (slices, maps, channels, interfaces e functions).
 
 One reason to switch to pointer semantics is if you need to share the slice for a
 decoding or unmarshaling operation. Using pointers for these types of operations
 are ok, but document this if it’s not obvious.
+Uma razão para mudar para a semântica de ponteiro é se você precisar compartilhar o `slice` para uma operação
+de decodificação ou desmarcação. Usar ponteiros para esses tipos de operações é aceitável, mas documente
+isso se não for óbvio.
```

Comment:
"decodificação ou desmarcação" -> "decoding ou unmarshal" or "decodificação e unmarshal", desmarcação is unusual in my opinion.